src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -34,19 +34,29 @@ public function normalize($object, $format = null, array $context = array())
         foreach ($object as $violation) {
             $violations[] = array(
                 'propertyPath' => $violation->getPropertyPath(),
-                'message' => $violation->getMessage(),
-                'code' => $violation->getCode(),
+                'type' => sprintf('urn:uuid:%s', $violation->getCode()),
+                'title' => $violation->getMessage(),
             );
             $propertyPath = $violation->getPropertyPath();
             $prefix = $propertyPath ? sprintf('%s: ', $propertyPath) : '';
             $messages[] = $prefix.$violation->getMessage();
         }
 
-        return array(
-            'title' => isset($context['title']) ? $context['title'] : 'An error occurred',
-            'detail' => $messages ? implode("\n", $messages) : '',
-            'violations' => $violations,
+        $result = array(
+            'type' => $context['type'] ?? 'https://symfony.com/doc/current/validation.html',
```

Comment:
I'd prefer a more generic URL (that could redirect to that page if we want to). Having a redirection also means we would be able to change the target very easily without breaking anything.

Reply:
`https://symfony.com/validation-error`?

Reply:
(Actually it's like a XML namespace, it doesn't even have to be dereferencable, but it's better from a DX POV)

Reply:
Let's use `https://symfony.com/validation-error`

Reply:
done